src/workerd/api/tests/transform-streams-test.js

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) 2024 Cloudflare, Inc.
```

Comment:
Non-blocking observation: This contains all the test cases from the corresponding internal file except for "Transform regression case". That last test case looks bizarre, but I think it's portable.